src/tree/taffy_tree/tree.rs

```diff
@@ -720,10 +912,4 @@ mod tests {
         );
         assert!(layout_result.is_ok());
     }
-
-    #[test]
-    fn measure_func_is_send_and_sync() {
-        fn is_send_and_sync<T: Send + Sync>() {}
-        is_send_and_sync::<MeasureFunc>();
-    }
```

Comment:
Please restore this test. This is part of our public API.

Reply:
Ohh I thought I'd fully removed the Send + Sync measurable... Anyway, putting it back :)